lib/client.js

```diff
@@ -129,10 +132,12 @@ var clientOptions = require('./client-options');
  * </p>
  * Default: false.
```

Comment:
`retryOnTimeout` was used before `onRequestError()` was added to the `RetryPolicy` prototype. Now it's maintained for historical reasons.

Could be useful is to have the `fetchSize`, though.